fem/bilinearform.cpp

```diff
@@ -992,6 +995,10 @@ void BilinearForm::EliminateVDofs(const Array<int> &vdofs_,
       mat_e = new SparseMatrix(height);
    }
 
+   vdofs_.HostRead();
+   mat->HostReadWriteI();
+   mat->HostReadWriteJ();
+   mat->HostReadWriteData();
```

Comment:
Ditto

Reply:
done in [caa79bf](https://github.com/mfem/mfem/pull/3127/commits/caa79bf80147c2b36b3f1070ec890221cbe2f11d).